app/assets/javascripts/discourse/app/controllers/discovery/topics.js

```diff
@@ -57,9 +57,11 @@ const controllerOpts = {
       return false;
     },
 
-    refresh() {
+    refresh(resetParams = true) {
```

Comment:
If the filter query param is "tracked" and you call refresh, then the query param is removed. I added this option, so that when dismissing unread with "tracked" set, it does not get unset.

Reply:
in this case I think @eviltrout prefers to go with `{ resetParams: true }`.

It makes the code mor extensible in the future and easier to follow when read, eg:

```javascript
this.refresh(true); // what does true means here?
this.refresh({ resetParams: true }); // very clear what true means here
```